app/Http/Controllers/Three/PostsController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Rogue\Http\Controllers\Three;
+
+use Rogue\Models\Post;
+use Illuminate\Http\Request;
+use Rogue\Services\PostService;
+use Rogue\Repositories\SignupRepository;
+use Rogue\Http\Transformers\PostTransformer;
+use Rogue\Http\Controllers\Api\ApiController;
+use Rogue\Http\Controllers\Traits\PostRequests;
+
+class PostsController extends ApiController
+{
+    use PostRequests;
```

Comment:
I know we use `PostRequests` to DRY up this code so we don't have the same thing in a bunch of different places, but I've found working with it confusing. I think we should all re-visit this and see if we still want it. Definitely not needed for this PR since Shae is out and of course we want to hear from her too!